### ISSUE TITLE
OK and Cancel buttons obstruct calendar days on large screens

### ISSUE THREAD

**Issue by pagefault** [misha130/datepicker-ionic2#17]
I am using the code from the example and when using a browser window in a desktop size rather than on a phone the two buttons at the bottom are on top of the last rows of days, making it not possible to select those.

![grafik](https://cloud.githubusercontent.com/assets/8460935/26168161/706d91ba-3b39-11e7-9f8d-efa441c79995.png)

**Comment by misha130** [comment]
Override the css with your own css.
```
.calendar-row { 
    padding-top: 6px!important;
    padding-bottom: 6px!important;
}
```

**Comment by misha130** [comment]
Fixed in 2.4.0
Thanks!
